src/site/docs/tutorials/get-started/index.markdown

```diff
@@ -171,11 +171,8 @@ in your Dart installation directory
 
 Dart Editor displays its Welcome Page
 in a tab in the **Editor pane**.
-The following diagram highlights
```

Comment:
I think we need to add back the diagram. It could be small or fuzzy, but it needs to show the basic parts that we refer to.